stack.yaml

```diff
@@ -20,7 +20,7 @@ packages:
 
 extra-deps:
 - git: https://github.com/serokell/cardano-sl
-  commit: aebb30b931d24386cdf959fd71c61cf01ba1edbf # ariadne branch
+  commit: a9a58706d4d05b1197751fae57d708a0f74bcc7e # ariadne branch
```

Comment:
Note: https://github.com/serokell/cardano-sl/pull/46 should be merged first, then I'll update this commit.